### PR TITLE
Updated examples from source

### LINKED ISSUE #2: Update UnicodeMath tests from source from Dec 18 2023
When we imported:
* https://github.com/MurrayIII/UnicodeMathML/blob/af8899f77e3ca5eba3738d243c7e13b796795343/dist/unicodemathml-integration.js

Now:
* https://github.com/MurrayIII/UnicodeMathML/blob/main/dist/unicodemathml-integration.js

Diff:
* https://github.com/MurrayIII/UnicodeMathML/compare/15e1bf625116e703f0b375f188b6aa43496ddddb..main#diff-f6164dc4e08c97f1e1d1d587f28dc884a2d0e788d90aa1f1feed393abe2125bd

### PR DESCRIPTION
**Pull request by suleman-uzair**
This PR updates examples from source mentioned in the [file](https://github.com/plurimath/unicodemath-tests/blob/main/unicodemath_tests.yaml).

closes #2

### CODE REVIEW

**Comment by suleman-uzair** [comment]
@ronaldtse Could you take a look at the PR and also give the access rights of the repo?
I forked it for now to create this PR.

**Comment by ronaldtse** [comment]
@suleman-uzair you now have access. Thanks!

File: unicodemath_tests.yaml [hunk 1]
```diff
@@ -1,28 +1,28 @@
 ---
 # This test suite is obtained from https://github.com/MurrayIII/UnicodeMathML/blob/main/dist/unicodemathml-integration.js
 # Courtesy of Sargent Murray III.
-last_updated: 2023-12-17T00:00:00Z
-sourced_from: https://github.com/MurrayIII/UnicodeMathML/commit/15e1bf625116e703f0b375f188b6aa43496ddddb
+last_updated: 2024-02-23T00:00:00Z
+sourced_from: https://github.com/MurrayIII/UnicodeMathML/blob/main/dist/unicodemathml-integration.js
```
Comment (ronaldtse) [inline unicodemath_tests.yaml]: Can we use the actual commit link because the “main” branch slides?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
Thank you @suleman-uzair !

**Pull request merged**